<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-09-17T12:17:54Z" name="Default suite" finished-at="2024-09-17T12:18:29Z" duration-ms="35698">
    <groups>
    </groups>
    <test started-at="2024-09-17T12:17:54Z" name="Default test" finished-at="2024-09-17T12:18:29Z" duration-ms="35698">
      <class name="com.AmazonAutomationProject.testCases.TestCase">
        <test-method is-config="true" signature="setup()[pri:0, instance:com.AmazonAutomationProject.testCases.TestCase@7975d1d8]" started-at="2024-09-17T17:47:54Z" name="setup" finished-at="2024-09-17T17:48:01Z" duration-ms="7462" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verifyTestAddToCart()[pri:0, instance:com.AmazonAutomationProject.testCases.TestCase@7975d1d8]" started-at="2024-09-17T17:48:01Z" name="verifyTestAddToCart" finished-at="2024-09-17T17:48:29Z" duration-ms="27570" status="FAIL">
          <exception class="java.lang.NullPointerException">
            <message>
              <![CDATA[Cannot invoke "org.openqa.selenium.WebElement.click()" because "visibleButton" is null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.NullPointerException: Cannot invoke "org.openqa.selenium.WebElement.click()" because "visibleButton" is null
	at com.AmazonAutomationProject.testCases.TestCase.verifyTestAddToCart(TestCase.java:39)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.NullPointerException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyTestAddToCart -->
        <test-method is-config="true" signature="tearDown()[pri:0, instance:com.AmazonAutomationProject.testCases.TestCase@7975d1d8]" started-at="2024-09-17T17:48:29Z" name="tearDown" finished-at="2024-09-17T17:48:29Z" duration-ms="603" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.AmazonAutomationProject.testCases.TestCase -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
